context.go

```diff
@@ -92,9 +91,9 @@ func (c *Context) LastError() error {
 
 	if err != nil {
 		var msg *C.char
+		defer C.free(unsafe.Pointer(msg))
```

Comment:
This is wrong, we don't own these strings its owned by the error message in C, we must not free it.